components/AllCoins.js

```diff
@@ -0,0 +1,65 @@
+import React, { useState, useEffect, useRef } from "react";
+
+import { Header } from "./Header";
+import { Wallet, WalletImage, WalletText, Container, CoinCardContainer, CoinCardTop, CoinCardBottom, CoinTitle, CoinInfo, CoinSymbol, ButtonContainer, BuyButton, SellButton } from "../styledcomponents/AllCoinsStyles";
```

Comment:
I have an ultrawide screen but still, I need to scroll to read all the imports 🙀 

```jsx
import { 
  Wallet, 
  WalletImage, 
  WalletText,
  Container,
  CoinCardContainer,
  CoinCardTop,
  CoinCardBottom,
  CoinTitle, CoinInfo,
  CoinSymbol,
  ButtonContainer,
  BuyButton,
  SellButton } from "../styledcomponents/AllCoinsStyles";
```
Just looks better and makes debugging much easier 😸